raft-tech/cfgov-refresh
# Jb snap logic ui

This PR will update the UI to only show Week Starting Balance, Week Ending Balance and SNAP Ending Balance (if SNAP benefits have been added).


## Changes

- New logic to handle rendering of SNAP ending balance



## How to test this PR

1. May require running ```./frontend.sh``` or at least ```gulp scripts:moneytools``` in order to test new features.

2.) Add starting balance, add SNAP benefit, see in calendar view that SNAP week ending balance is visible.

## Screenshots
#### Old
<img width="839" alt="Screen Shot 2020-06-16 at 9 05 06 AM" src="https://user-images.githubusercontent.com/21065244/84959062-dceb4300-b0bb-11ea-9949-be9ae5068762.png">

#### New
<img width="671" alt="Screen Shot 2020-06-16 at 4 36 30 PM" src="https://user-images.githubusercontent.com/21065244/84959093-ed032280-b0bb-11ea-8da3-2e33dfafd0cd.png">




## Notes and todos

- This PR includes rebase of  ```my-money-calendar``` which is our new "source of truth" branch for Raft focused development. After merging this to ```my-money-calendar```, we should be caught up with latest features.
